ingest-app/src/cmr/ingest/api/ingest.clj

```diff
@@ -456,13 +455,11 @@
   (generate-provider-task-status-response
    headers
    {:status 200
-    :task-status "Partial Fail"
-    :status-message "The following collections had errors: C-1, C-2"
+    :task-status "COMPLETE"
```

Comment:
I see you are pushing the partial failure status down the status-message. I guess it is OK.